fetch.php

```diff
@@ -1,5 +1,7 @@
-<?
+<?php

```

Comment:
Looks like `fetch.php` is missing the closing tag too.

Reply:
End tags are not required anymore unless you are inserting a php snippet in the middle of another file (like an html file).